cardano-db/src/Cardano/Db/Query.hs

```diff
@@ -390,12 +400,12 @@ querySelectCount predicate = do
             pure countRows
   pure $ maybe 0 unValue (listToMaybe xs)
 
-querySlotHash :: MonadIO m => SlotNo -> ReaderT SqlBackend m (Maybe (SlotNo, ByteString))
+querySlotHash :: MonadIO m => SlotNo -> ReaderT SqlBackend m [(SlotNo, ByteString)]
```

Comment:
Since we can have multiple blocks in the same slot (EBBs), we return all of them and allow the caller to decide.